app/controllers/webhooks_controller.rb

```diff
@@ -36,4 +36,27 @@ def gocardless
 
     head(:ok)
   end
+
+  def adyen
+    signature = adyen_params.dig("additionalData", "hmacSignature")
+
+    result = PaymentProviders::AdyenService.new.handle_incoming_webhook(
+      organization_id: params[:organization_id],
+      body: adyen_params
+    )
+
+    unless result.success?
+      if result.error.code == 'webhook_error'
+        return head(:bad_request)
+      end
+
+      result.raise_if_error!
+    end
+
+    render json: "[accepted]"
```

Comment:
Style/MethodCallWithArgsParentheses: Use parentheses for method calls with arguments. (https://rubystyle.guide#method-invocation-parens)